apps/evm/src/ui/pool/Steer/SteerLiquidityManagement/Add/SteerPositionAddReviewModal.tsx

```diff
@@ -119,7 +120,9 @@ export const SteerPositionAddReviewModal: FC<SteerPositionAddReviewModalProps> =
     const slippagePercent = useMemo(() => {
       return new Percent(
         Math.floor(
-          +(slippageTolerance === 'AUTO' ? '0.1' : slippageTolerance) * 100,
+          +(slippageTolerance === 'AUTO'
```

Comment:
Still seems weird to be repeating this here and newing up new percent object in each of these views.